### PR TITLE
Update documents and example name for #4263

### LINKED ISSUE #4299: Follow up fix for #4263
See comments in https://github.com/vega/vega-lite/pull/4263

### PR DESCRIPTION
**Pull request by chanwutk**
Fix #4299

### CODE REVIEW

File: site/docs/mark/errorband.md [hunk 1]
```diff
@@ -22,12 +23,14 @@ Error band in Vega-Lite can either be used to aggregate raw data or directly vis
 To create an error band, set `mark` to `"errorband"`.
 
 ## Documentation Overview
+
 {:.no_toc}
 
 - TOC
-{:toc}
+  {:toc}
```
Comment (domoritz) [inline site/docs/mark/errorband.md]: why?

File: site/docs/mark/errorbar.md [hunk 2]
```diff
@@ -22,12 +23,14 @@ Error bars in Vega-Lite can either be used to aggregate raw data or directly vis
 To create an error bar, set `mark` to `"errorbar"`.
 
 ## Documentation Overview
+
 {:.no_toc}
 
 - TOC
-{:toc}
+  {:toc}
```
Comment (domoritz) [inline site/docs/mark/errorbar.md]: Why

File: site/docs/mark/errorband.md (hunk 1 above)
Comment (chanwutk) [reply site/docs/mark/errorband.md]: This is what prettier does for me. Should I revert back?

**Comment by kanitw** [comment]
> Hmm, this PR does not seem to delete the existing SVGs/compiled specs, does it?

I think the script do not delete them by default.  So you have to manually delete them.  Or even better, update the script to do so.

**Comment by domoritz** [comment]
Yep, let's fix the script. I'm running an experiment in https://github.com/vega/vega-lite/pull/4312 right now.

**Comment by domoritz** [comment]
@chanwutk Please update the script to generate specs/svgs to either delete them or create an issue to fix this later.

**Comment by kanitw** [comment]
The tricky part is that we only re-run SVGs when Vega changes.  

So we might want to
1) Always delete all Vega
2) Check if any remaining SVGs do not Vega source and delete them.

File: site/docs/mark/errorband.md (hunk 1 above)
Comment (domoritz) [reply site/docs/mark/errorband.md]: Please rebase so you have the right format.

File: site/docs/mark/errorbar.md (hunk 2 above)
Comment (domoritz) [reply site/docs/mark/errorbar.md]: Still needs to be reverted

File: scripts/check-and-fix.sh [hunk 3]
```diff
@@ -46,17 +46,17 @@ fi
 # Commit examples if outdated
 
 # Note: we need to add all files first so that new files are included in `git diff --cached` too.
-git add ./examples/compiled/vega_version ./examples/compiled/*.vg.json ./examples/compiled/*.svg ./examples/specs/normalized/*.vl.json
+git add ./examples/compiled/vega_version './examples/compiled/*.vg.json' './examples/compiled/*.svg' './examples/specs/normalized/*.vl.json'
```
Comment (kanitw) [inline scripts/check-and-fix.sh]: Why are these script fix a part of this follow fix?  

If they're not the same issue, I don't think we should merge them in the same PR?

Comment (kanitw) [reply scripts/check-and-fix.sh]: NVM -- it's good to combine them given they're a good test bed for this issue.

**Comment by kanitw** [comment]
Thank you :)

**Pull request merged**